code/utils/scene_slicer.py

```diff
@@ -36,31 +34,28 @@ def generate_scene_desc_dict(self):
                 scene_time = int(float(row['seconds']))
                 self.scene_desc[scene_time] = (row['day-night'] == "DAY",
```

Comment:
So we can just make this a tuple of 0 or 1, so we don't have to convert the boolean into a number later.